### PR TITLE
Added Synthwave '84 theme

### PR DESCRIPTION
**Pull request by themarcba**
I added the Synthwave '84 VS Code theme, originally by Robb Owen (I acquired his permission before publishing it).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,4 +71,7 @@ To use one of the themes, just include the theme's CSS file in your page. Exampl
 [![a11y Dark](screenshots/prism-a11y-dark.png)](themes/prism-a11y-dark.css)
 
 * [__Dracula__](themes/prism-dracula.css) (by [Byverdu](https://github.com/byverdu))<br />
-[![a11y Dark](screenshots/prism-dracula.png)](themes/prism-dracula.css)
\ No newline at end of file
+[![a11y Dark](screenshots/prism-dracula.png)](themes/prism-dracula.css)
+
+* [__Synthwave '84__](themes/prism-synthwave84.css) (originally by [Robb Owen](https://github.com/robb0wen), adapted by [Marc Backes](https://github.com/themarcba))<br />
+[![CB](screenshots/prism-synthwave84.png)](themes/prism-synthwave84.css)
```
Comment (mAAdhaTTah) [inline README.md]:
```suggestion
[![Synthwave '84](screenshots/prism-synthwave84.png)](themes/prism-synthwave84.css)
```

File: themes/prism-synthwave84.css [hunk 2]
```diff
@@ -0,0 +1,142 @@
+/*
+ * Synthwave '84 Theme originally by Robb Owen [@Robb0wen] for Visual Studio Code
+ * Demo: https://marc.dev/demo/prism-synthwave84
+ *
+ * Ported for PrismJS by Marc Backes [@themarcba]
+ */
+
+ code[class*="language-"],
+ pre[class*="language-"] {
+     color: #f92aad;
```
Comment (mAAdhaTTah) [inline themes/prism-synthwave84.css]: We use tabs.

File: themes/prism-synthwave84.css [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+     font-style: italic;
+ }
+ 
+ .token.entity {
+     cursor: help;
+ }
+ 
+ .token.inserted {
+     color: green;
+ }
```
Comment (mAAdhaTTah) [inline themes/prism-synthwave84.css]: Needs a newline @ end of file.

**Comment by mAAdhaTTah** [comment]
Couple of minor nits, but this is sick! Thanks for the contribution!

File: themes/prism-synthwave84.css (hunk 3 above)
Comment (themarcba) [reply themes/prism-synthwave84.css]: Thanks for reviewing the pull request. And this is terribly embarrassing, but I've never created a pull request before, therefore I don't know how this all works...

You pointed out some flaws in the contribution. Do I need to change those, or did you already do the changes?

Comment (mAAdhaTTah) [reply themes/prism-synthwave84.css]: Yeah, make the suggested changes, add them as commits to the branch, and push that branch to your remote. Then the PR will be updated with them.

Comment (RunDevelopment) [reply themes/prism-synthwave84.css]: @themarcba This still hasn't been fixed, has it?

File: README.md (hunk 1 above)
Comment (ahmadawais) [reply README.md]: The screenshot is also bigger than the others. 250px x 180px seems to be the average.

Comment (mAAdhaTTah) [reply README.md]: @ahmadawais You're not wrong but I'm not going to block this PR over it.

File: themes/prism-synthwave84.css (hunk 3 above)
Comment (mAAdhaTTah) [reply themes/prism-synthwave84.css]: @RunDevelopment it is.

**Pull request merged**